mentat/parsers/block_parser.py

```diff
@@ -167,12 +167,12 @@ def _special_block(
         if deserialized_json.action == _BlockParserAction.Delete:
             replacements.append(Replacement(starting_line, ending_line, []))
         file_edit = FileEdit(
-            git_root / deserialized_json.file,
```

Comment:
So the model is going to be outputting paths relative to the cwd instead of git_root now, right? That sounds good to me, just wanted to confirm.

Reply:
Yup!